### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Prakash-Haldar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,88 @@ This project is a starting point for a Flutter application.
...
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and make it better?
+To start contributing, follow the below guidelines:
+
+**1.** Fork [this](https://github.com/geekymeeky/chitchat) repository.
+
+**2.** Clone your forked copy of the project.
+
+```
+git clone --depth 1 https://github.com/<your_user_name>/chitchat.git
```
Comment (geekymeeky) [inline README.md]: Make use of this. It becomes more readable and accessible as a result.
```bash
git clone https://github.com/<your_user_name>/chitchat.git
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,88 @@ This project is a starting point for a Flutter application.
...
+```
+
+<img src="./readme_assets/CloneRepo.png" width="500">
```
Comment (geekymeeky) [inline README.md]: This image isn't rendering.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,88 @@ This project is a starting point for a Flutter application.
...
+
+**11.** Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**12.** To create a pull request, click on `compare and pull requests. Please ensure you compare your feature branch to the desired branch of the repo you are supposed to make a PR to.
+
+<img src="./readme_assets/compare_pr.png" width=600>
```
Comment (geekymeeky) [inline README.md]: This image isn't rendering.

File: README.md [hunk 4]
```diff
@@ -8,6 +8,88 @@ This project is a starting point for a Flutter application.
...
+
+**13.** Add appropriate title and description to your pull request explaining your changes and efforts done.
+
+**14.** Click on `Create Pull Request`.
+<img src="./readme_assets/CreatePR.png" width=600>
```
Comment (geekymeeky) [inline README.md]: This image isn't rendering.

**Review by geekymeeky — CHANGES_REQUESTED** [review]

**Comment by Prakash-Haldar** [comment]
@geekymeeky I Have done all the Changes . Now it works Good . Please See to it

**Review by geekymeeky — APPROVED** [review]

**Comment by Prakash-Haldar** [comment]
Please Send the google form of DevIncept @geekymeeky

**Comment by geekymeeky** [comment]
> Please Send the google form of DevIncept @geekymeeky

@Prakash-Haldar Check the Slack channel

**Comment by geekymeeky** [comment]
Fixes #12

**Comment by Prakash-Haldar** [comment]
Please Send Me the Link Of Slack Channel

**Pull request merged**